src/views/template/warehouse/warehouse.html

```diff
@@ -9,13 +9,13 @@
         </span>
     </div>
     <div>
-        <div class="progress no-border-radius">
+        <div class="progress no-border-radius margin-bottom-0">
```

Comment:
I don't have the scrollbar (yet), but I'll take your word for it. We can always revise later.

This can stay!